enrichment_method.py

```diff
@@ -17,6 +17,11 @@ def _get_traceback(exception):
         )
     )
 
+def get_environment_variable(variable):

```

Comment:
This is a good first effort  - we've talked about before we should `marshmallow` all the things though.